lib/src/speed_dial.dart

```diff
@@ -217,23 +221,44 @@ class _SpeedDialState extends State<SpeedDial> with SingleTickerProviderStateMix
       curve: widget.curve,
     );
 
-    return Positioned(
-      bottom: widget.marginBottom - 16,
-      right: widget.marginRight - 16,
-      child: Container(
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.end,
-          crossAxisAlignment: CrossAxisAlignment.end,
-          children: List.from(fabChildren)
-            ..add(
-              Container(
-                margin: EdgeInsets.only(top: 8.0, right: 2.0),
-                child: animatedFloatingButton,
-              ),
+    switch (widget.orientation) {
+      case SpeedDialOrientation.Up:
+        return Positioned(
+          bottom: widget.marginBottom - 16,
+          right: widget.marginRight - 16,
+          child: Container(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.end,
+              crossAxisAlignment: CrossAxisAlignment.end,
+              children: List.from(fabChildren)
+                ..add(Container(
+                  margin: EdgeInsets.only(top: 8.0, right: 2.0),
+                  child: animatedFloatingButton,
+                )),
             ),
-        ),
-      ),
-    );
+          ),
+        );
+        break;
+      case SpeedDialOrientation.Down:
+        return Positioned(
+          top: MediaQuery.of(context).size.height - 56 - 8 - (widget.marginBottom - 16),
+          right: widget.marginRight - 16,
+          child: Container(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.end,
+              crossAxisAlignment: CrossAxisAlignment.end,
+              children: List.from(fabChildren.reversed)
+                ..insert(
+                    0,
+                    Container(
+                      margin: EdgeInsets.only(top: 8.0, right: 2.0),
```

Comment:
When in orientation .down this 8px top margin should be a bottom one.

Reply:
Done.